extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,510 @@
+<!-- omit in toc -->
+# EXT_feature_metadata
+
+**Version 1.0.0** [TODO: date]
+
+<!-- omit in toc -->
+## Contributors
+
+* Peter Gagliardi, Cesium
+* Sean Lilley, Cesium
+* Bao Tran, Cesium
+* Sam Suhag, Cesium
+* Samuel Vargas, Cesium
+* Patrick Cozzi, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
+
+Written against the glTF 2.0 specification.
+
+Optionally extends the [`EXT_mesh_gpu_instancing` extension](../../EXT_mesh_gpu_instancing/README.md).
+
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is optional, meaning it should be placed in the `extensionsUsed` list, but not in the `extensionsRequired` list.
+
+<!-- omit in toc -->
+## Contents
+
+- [Overview](#overview)
+- [Feature Identification](#feature-identification)
+  - [Feature ID Vertex Attributes](#feature-id-vertex-attributes)
+    - [Feature ID Accessors](#feature-id-accessors)
+    - [Implicit Feature IDs](#implicit-feature-ids)
+  - [Feature ID Textures](#feature-id-textures)
+  - [Feature ID Instance Attributes](#feature-id-instance-attributes)
+- [Feature Metadata](#feature-metadata)
+  - [Schemas](#schemas)
+  - [Feature Tables](#feature-tables)
+  - [Feature Textures](#feature-textures)
+  - [Statistics](#statistics)
+- [Examples](#examples)
+
+## Overview
+
+This extension allows offline batching of heterogeneous 3D features, such as different buildings in a city, for efficient streaming to a client for rendering and interaction. Efficiency comes from transferring multiple features in a single request and rendering them in the least number of draw calls necessary.
+
+Feature IDs enable individual features to be identified and updated at runtime, e.g., show/hide, highlight color, etc. Feature IDs may be assigned on a per-vertex, per-texel, or per-instance basis.
```

Comment:
This first sentence feels a bit awkward, maybe reword. Also I'd say use `e.g.` OR use `etc.` but both together feels a bit much.
```suggestion
Feature IDs enable individual features to be identified and updated at runtime, e.g., show/hide, highlight color, etc. Feature IDs may be assigned on a per-vertex, per-texel, or per-instance basis.
```